README.md

```diff
@@ -38,11 +38,13 @@ The primary ETL script is [`load_results_into_survey_project.R`](load_results_in
 
 To build the image and run the report using docker within the project directory do:
 
-`docker build -t <image_name> .`
+`docker build -t fr_covidata_engine_all .`
 
-and run the report using docker within the project directory like this:
+and run the script using docker with a command something like this:
 
-`docker run --env-file .env -v path/from/host:/home/fr_covidata_engine <image_name>`
+`docker run --rm --env-file <path_to_dir_full_of_env_files>/fr_dev.env fr_covidata_engine_all Rscript load_results_into_survey_project.R`
```

Comment:
Did you create a folder with env files and where is fr_dev.env?

Reply:
The folder with env files is the root of the repo.  create `<path_to_dir_full_of_env_files>` is a local implementation detail.

What if I said `example.env` instead of `fr_dev.env`?

Reply:
Oh so you're saying that .env and example.env is in the project folder aka root of repo? I think .env works since the user must create a .env file to run the container and we listed the instructions for creating the .env in the configuration.